packages/ingest/src/granule.ts

```diff
@@ -410,12 +415,73 @@ export async function moveGranuleFiles(
       }
     }
   );
-
   await Promise.all(moveFileRequests);
-
   return movedGranuleFiles;
 }
 
+export async function moveGranuleFile(
```

Comment:
This method is a port of the existing moveGranuleFiles logic, to be a single file operation with postgres updates added in by necessity to tie the S3 move to the database update in a transaction.